frontend/src/common/components/Organism/Header.vue

```diff
@@ -28,8 +28,7 @@ const modalContent = ref<'login' | 'signup' | 'password' | 'close'>('close')
       <div class="flex w-full max-w-7xl items-center justify-between gap-5">
         <div class="flex w-1/2 min-w-fit items-center justify-between gap-8">
           <RouterLink to="/" class="flex items-center gap-1">
```

Comment:
```suggestion
          <RouterLink to="/">
```

이제 flex 지워도 될 것 같아요!

Reply:
넵!